tls/s2n_security_policies.c

```diff
@@ -165,54 +166,104 @@ const struct s2n_security_policy security_policy_cloudfront_upstream_tls12 = {
     .ecc_preferences = &s2n_ecc_preferences_20140601,
 };
 
+/* CloudFront viewer facing */
 const struct s2n_security_policy security_policy_cloudfront_ssl_v_3 = {
     .minimum_protocol_version = S2N_SSLv3,
     .cipher_preferences = &cipher_preferences_cloudfront_ssl_v_3,
     .kem_preferences = &kem_preferences_null,
-    .signature_preferences = &s2n_signature_preferences_20140601,
-    .ecc_preferences = &s2n_ecc_preferences_20140601,
+    .signature_preferences = &s2n_signature_preferences_20200207,
```

Comment:
why are those changed?

Reply:
this is to update the signature preference that includes x25519 which is needed for support majority of TLS 1.3 clients

Reply:
correction - signature preference is required to enable rsa pss signing required for TLS 1.3